test/setup/iam.tf

```diff
@@ -0,0 +1,73 @@
+/**
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+locals {
+  int_required_roles = [
+    "roles/cloudfunctions.developer",
+    "roles/storage.admin",
+    "roles/compute.admin",
+    "roles/pubsub.admin",
+    "roles/logging.configWriter",
+    "roles/source.admin",
+    "roles/iam.serviceAccountUser",
+  ]
+
+  int_required_folder_roles = [
```

Comment:
Why do we need a folder for this module to be tested? I'd think the integration config for this module could be very similar to that for the VM module: https://github.com/terraform-google-modules/terraform-google-vm/blob/master/test/setup/iam.tf

Reply:
Same thing. I just copied that. I'll clean that up.

Reply:
I cleaned this too.